doc/ui_functions.md

```diff
@@ -44,11 +43,11 @@ The tooltip is shown when hover on feature.
 
 The tooltip is not shown when hover on feature.
 
+
 ## cartodb.geo.ui.Zoom
 
 Shows the zoom control:
 
-<div class="code-title">cartodb.geo.ui.Zoom</div>
 ```javascript
 vis.addOverlay({ type: 'zoom' });
 ```
```

Comment:
Even if it wasn't there and it's pretty obvious, we might want to add a description for `cartodb.geo.ui.Zoom.show()` and `cartodb.geo.ui.Zoom.hide()`.